src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -69,14 +69,21 @@ class RoddyIOHelperMethods {
 
         @Override
         void compressFile(File from, File to, File workingDirectory = null) {
-            compressDirectory(from, to, workingDirectory);
+            try {
+                compressDirectory(from, to, workingDirectory);
+            } catch (Exception ex) {
+                throw new IOException("Could not zip file ${from} to zip archive ${to}", ex)
```

Comment:
You catch exceptions already in compressDirectory, why are you catching  it again?